react/src/components/Location.tsx

```diff
@@ -15,225 +15,164 @@ enum ComponentLocationState {
   EnterManually
 }
 
-interface State {
-  componentLocationState: ComponentLocationState;
-  manualAddress: string | undefined;
-  locationError: string | undefined;
-}
+const ERROR_MESSAGE = `We weren't able to find that location! Please try again with a different address.`;
 
-class Location extends React.Component<
```

Comment:
Needed to refactor this component from a class to a function component to use toastify -- in general, it would be nice to do a pr where I just do a general refactoring. there are a lot of advantages, not least that some libs won't work for class components.

Reply:
Agree, this is painful to look at class components every time I jump in to editing these. I'll take a look at doing a quick refactor of all these things in the near future.